fineract-loan/src/main/java/org/apache/fineract/portfolio/repaymentwithpostdatedchecks/domain/PostDatedChecks.java

```diff
@@ -40,7 +39,7 @@ public class PostDatedChecks extends AbstractPersistableCustom {
     @ManyToOne(optional = false)
     @JoinColumn(name = "loan_id", referencedColumnName = "id", nullable = false)
     private Loan loan;
-    @OneToOne(optional = false)
+    @ManyToOne(optional = false)
```

Comment:
This was the problem? :o

Reply:
Yeah... because the other side is `@OneToMany`